### ISSUE TITLE
Hotplug not working correctly

### ISSUE THREAD

**Issue by foxbunny** [Outernet-Project/orx-rpi#8]
On Pi2, it has been reported that hotplug does not work correctly.

`udevadm` shows the following env vars:

```
calling: test
version 3.1.2
This program is for debugging only, it does not run any program
specified by a RUN key. It may show incorrect results, because
some values may be different, or not available at a simulation run.
/etc/udev/hwdb.bindoes not exist, please run udevadm hwdb --update
Load module index
timestamp of '/etc/udev/rules.d' changed
timestamp of '/lib/udev/rules.d' changed
Reading rules file: /lib/udev/rules.d/42-usb-hid-pm.rules
Reading rules file: /etc/udev/rules.d/50-dvb.rules
Reading rules file: /etc/udev/rules.d/50-storage.rules
Reading rules file: /lib/udev/rules.d/50-udev-default.rules
Reading rules file: /etc/udev/rules.d/50-wifi.rules
Reading rules file: /lib/udev/rules.d/60-block.rules
Reading rules file: /lib/udev/rules.d/60-cdrom_id.rules
Reading rules file: /lib/udev/rules.d/60-drm.rules
Reading rules file: /lib/udev/rules.d/60-evdev.rules
Reading rules file: /lib/udev/rules.d/60-persistent-alsa.rules
Reading rules file: /lib/udev/rules.d/60-persistent-input.rules
Reading rules file: /lib/udev/rules.d/60-persistent-storage-tape.rules
Reading rules file: /lib/udev/rules.d/60-persistent-storage.rules
Reading rules file: /lib/udev/rules.d/60-persistent-v4l.rules
Reading rules file: /lib/udev/rules.d/60-serial.rules
Reading rules file: /lib/udev/rules.d/61-accelerometer.rules
Reading rules file: /lib/udev/rules.d/64-btrfs.rules
Reading rules file: /lib/udev/rules.d/70-mouse.rules
Reading rules file: /lib/udev/rules.d/70-touchpad.rules
Reading rules file: /lib/udev/rules.d/75-net-description.rules
Reading rules file: /lib/udev/rules.d/75-probe_mtd.rules
Reading rules file: /lib/udev/rules.d/78-sound-card.rules
Reading rules file: /lib/udev/rules.d/80-drivers.rules
Reading rules file: /lib/udev/rules.d/80-net-name-slot.rules
rules contain 24576 bytes tokens (2048 * 12 bytes), 7298 bytes strings
1022 strings (12688 bytes), 636 de-duplicated (5777 bytes), 387 trie nodes used
RUN '/sbin/hotplug.storage' /etc/udev/rules.d/50-storage.rules:1
GROUP 6 /lib/udev/rules.d/50-udev-default.rules:55
LINK 'disk/by-id/usb-hp_v165w_AA04012700008389-0:0-part1' /lib/udev/rules.d/60-persistent-storage.rules:36
LINK 'disk/by-path/platform-bcm2708_usb-usb-0:1.2:1.0-scsi-0:0:0:0-part1' /lib/udev/rules.d/60-persistent-storage.rules:55
IMPORT builtin 'blkid' /lib/udev/rules.d/60-persistent-storage.rules:65
probe /dev/sda1 raid offset=0
LINK 'disk/by-uuid/057E-20A3' /lib/udev/rules.d/60-persistent-storage.rules:68
LINK 'disk/by-label/EXT_STORE' /lib/udev/rules.d/60-persistent-storage.rules:69
handling device node '/dev/sda1', devnum=b8:1, mode=0660, uid=0, gid=6
preserve permissions /dev/sda1, 060660, uid=0, gid=6
preserve already existing symlink '/dev/block/8:1' to '../sda1'
found 'b8:1' claiming '/run/udev/links/\x2fdisk\x2fby-id\x2fusb-hp_v165w_AA04012700008389-0:0-part1'
creating link '/dev/disk/by-id/usb-hp_v165w_AA04012700008389-0:0-part1' to '/dev/sda1'
preserve already existing symlink '/dev/disk/by-id/usb-hp_v165w_AA04012700008389-0:0-part1' to '../../sda1'
found 'b8:1' claiming '/run/udev/links/\x2fdisk\x2fby-label\x2fEXT_STORE'
creating link '/dev/disk/by-label/EXT_STORE' to '/dev/sda1'
preserve already existing symlink '/dev/disk/by-label/EXT_STORE' to '../../sda1'
found 'b8:1' claiming '/run/udev/links/\x2fdisk\x2fby-path\x2fplatform-bcm2708_usb-usb-0:1.2:1.0-scsi-0:0:0:0-part1'
creating link '/dev/disk/by-path/platform-bcm2708_usb-usb-0:1.2:1.0-scsi-0:0:0:0-part1' to '/dev/sda1'
preserve already existing symlink '/dev/disk/by-path/platform-bcm2708_usb-usb-0:1.2:1.0-scsi-0:0:0:0-part1' to '../../sda1'
found 'b8:1' claiming '/run/udev/links/\x2fdisk\x2fby-uuid\x2f057E-20A3'
creating link '/dev/disk/by-uuid/057E-20A3' to '/dev/sda1'
preserve already existing symlink '/dev/disk/by-uuid/057E-20A3' to '../../sda1'
created db file '/run/udev/data/b8:1' for '/devices/platform/bcm2708_usb/usb1/1-1/1-1.2/1-1.2:1.0/host0/target0:0:0/0:0:0:0/block/sda/sda1'
ACTION=add
DEVLINKS=/dev/disk/by-id/usb-hp_v165w_AA04012700008389-0:0-part1 /dev/disk/by-label/EXT_STORE /devdisk/by-path/platform-bcm2708_usb-usb-0:1.2:1.0-scsi-0:0:0:0-part1 /dev/disk/by-uuid/057E-20A3
DEVNAME=/dev/sda1
DEVPATH=/devices/platform/bcm2708_usb/usb1/1-1/1-1.2/1-1.2:1.0/host0/target0:0:0/0:0:0:0/block/sda/sda1
DEVTYPE=partition
ID_BUS=usb
ID_FS_LABEL=EXT_STORE
ID_FS_LABEL_ENC=EXT_STORE
ID_FS_TYPE=vfat
ID_FS_USAGE=filesystem
ID_FS_UUID=057E-20A3
ID_FS_UUID_ENC=057E-20A3
ID_FS_VERSION=FAT32
ID_INSTANCE=0:0
ID_MODEL=v165w
ID_MODEL_ENC=v165w\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20
ID_MODEL_ID=5307
ID_PART_ENTRY_DISK=8:0
ID_PART_ENTRY_NUMBER=1
ID_PART_ENTRY_OFFSET=2048
ID_PART_ENTRY_SCHEME=dos
ID_PART_ENTRY_SIZE=15657985
ID_PART_ENTRY_TYPE=0xb
ID_PART_ENTRY_UUID=000ab8ec-01
ID_PART_TABLE_TYPE=dos
ID_PART_TABLE_UUID=000ab8ec
ID_PATH=platform-bcm2708_usb-usb-0:1.2:1.0-scsi-0:0:0:0
ID_PATH_TAG=platform-bcm2708_usb-usb-0_1_2_1_0-scsi-0_0_0_0
ID_REVISION=1100
ID_SERIAL=hp_v165w_AA04012700008389-0:0
ID_SERIAL_SHORT=AA04012700008389
ID_TYPE=disk
ID_USB_DRIVER=usb-storage
ID_USB_INTERFACES=:080650:
ID_USB_INTERFACE_NUM=00
ID_VENDOR=hp
ID_VENDOR_ENC=hp\x20\x20\x20\x20\x20\x20
ID_VENDOR_ID=03f0
MAJOR=8
MINOR=1
SUBSYSTEM=block
USEC_INITIALIZED=6356122421
run: '/sbin/hotplug.storage'
Unload module index
```

**Comment by foxbunny** [comment]
This is probably fixed in 3364c8617b78fdbe251567de5d18ddc1a8839d1a. Needs confirmation.

**Comment by foxbunny** [comment]
It appears 3364c86 is unrelated.
